src/jit/importer.cpp

```diff
@@ -14819,6 +14819,11 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 }
                 op1->gtFlags |= GTF_EXCEPT;
 
+                if (prefixFlags & PREFIX_UNALIGNED)
+                {
+                    op1->gtFlags |= GTF_IND_UNALIGNED;
+                }
+
```

Comment:
Ah, ok. It turns out `GTF_BLK_UNALIGNED` is exactly the same as `GTF_IND_UNALIGNED`, by design. So the code here is ok.

> BTW, could you let me know if there are any documents about GenTree?

Unfortunately, there is not very much documentation except for gentree.h itself. We could do better. I'd certainly like the comments for each GenTree class to be more verbose.